pkgs/applications/audio/klystrack/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub
+, SDL2, SDL2_image
+, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "klystrack";
+  version = "1.7.6";
+
+  src = fetchFromGitHub {
+    owner = "kometbomb";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "1h99sm2ddaq483hhk2s3z4bjbgn0d2h7qna7l7qq98wvhqix8iyz";
+  };
+
+  buildInputs = [
+    SDL2.dev SDL2_image
```

Comment:
Why `dev` here, nix should have get the correct output?

Reply:
`SDL2.dev` provides `sdl2-config` which is required because there's no way to specify sdl2 paths: https://github.com/kometbomb/klystrack/issues/260

Reply:
I could still build this locally when it's just `SDL2`.

Reply:
Cool. So https://github.com/NixOS/nixpkgs/issues/58177 is actually fixed.